Pipes - <Roxanne Agerone> - <Calculator>

```diff
@@ -0,0 +1,83 @@
+# adds with  add and +
+# subtracts with subtract and -
+# adds with multiply and *
+ # adds numbers with divide and /
+ # handles divide when attempting to divide by 0
+ # handles erroneous input
+
+ def number_question
+   numbers = []
+   puts "give me two numbers"
+       num = gets.chomp.to_f
+       numbers.push(num)
```

Comment:
Watch your indentation here - lines 11 and 12 should be at the same level as line 10.